src/service/stripe-service.ts

```diff
@@ -46,6 +48,17 @@ export default class StripeService {
     this.logger = getLogger('StripeController');
   }
 
+  /**
+   * Topup should be at least 10 euros or the users negative balance.
+   * @param balance
+   * @param request
+   */
+  public static validateStripeRequestAmount(balance: BalanceResponse, request: StripeRequest): boolean {
+    // Check if top up is enough
+    if (request.amount.amount > (process.env.MIN_TOPUP || 1000)) return true;
```

Comment:
Shouldn't this be greater than or equal? Because now you cannot top up with 10 euros exactly if you don't have a debt.